soweego/importer/musicbrainz_dump_extractor.py

```diff
@@ -247,6 +270,54 @@ def _artist_generator(self, dump_path):
                             current_entity, MusicbrainzBandEntity, aliases[artist['id']]):
                         yield alias
 
+    def _artist_band_relationship_generator(self, dump_path):
```

Comment:
It seems you have implemented the artist -> band relationship, but what about the inverse?

Reply:
I cannot see your point. There's a table of couples (artist, bands). If I query for a band I can get all its artists, and viceversa.

Reply:
N x M rows is the worst case scenario. It is also very unlikely.
On average an artist has two/three bands and a band has about 5 people. There are many solo artists, too.
On MusicBrainz, which counts about a million entities, the relationships were about 300 000.

Anyway this is almost the standard way of implementing an N x N relationship.

Is said almost because we are already saving rows in my implementation. We are not using FOREIGN KEYS because our entities table is denormalized. This means that with a standard implementation we would have to relate each artist alias to each band alias and so on.
Using the ID shared among the aliases in the relation, lets us save rows